GetDisplayUserInfo1/GetDisplayUserInfo1/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GetDisplayUserInfo1
+{
+    class Program
+    {        
+            static string getUserName(string question)
+        {
+            Console.WriteLine(question);
+            string answer = Console.ReadLine();
+            return answer;
+        }              
+        static int getUserNumber(string question)
+        {
+            Console.WriteLine(question);
+            string response = Console.ReadLine();
+            int answer = Int32.Parse(response);
+            return answer;
+        }
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, I would like some basic information from you.");
+            string firstName = getUserName("What is your first name?");
+            string lastName = getUserName("What is your last name?");  
+                                                           
+            int favoriteNumber = getUserNumber("What is your favorite number?");                      
+            int petCount = getUserNumber("How many pets do you have?");
+
+            if (petCount > 1)
+            {
+                List<string> petName = new List<string>();
```

Comment:
It would probably be good to name this variable with a plural since it's a list, such as `petNames`